src/parse-decimal-number.coffee

```diff
@@ -19,9 +19,9 @@ module.exports = (value,inOptions,enforceGroupSize=true)->
   pattern = patterns[patternIndex]
   if not pattern
     if enforceGroupSize
-      pattern = patterns[patternIndex] =  new RegExp ('^\\s*(-?(?:(?:\\d{1,3}(?:\\' + thousands + '\\d{3})+)|\\d*))(?:\\' + decimal + '(\\d*))?\\s*$')
+      pattern = patterns[patternIndex] =  new RegExp ('^\\s*([+\-]?(?:(?:\\d{1,3}(?:\\' + thousands + '\\d{3})+)|\\d*))(?:\\' + decimal + '(\\d*))?\\s*$')
     else
-      pattern = patterns[patternIndex] =  new RegExp ('^\\s*(-?(?:(?:\\d{1,3}(?:\\' + thousands + '\\d{1,3})+)|\\d*))(?:\\' + decimal + '(\\d*))?\\s*$')
+      pattern = patterns[patternIndex] =  new RegExp ('^\\s*([+\-]?(?:(?:\\d{1,3}(?:\\' + thousands + '\\d{1,3})+)|\\d*))(?:\\' + decimal + '(\\d*))?\\s*$')
```

Comment:
I could split the pattern over multiple lines, but I assume you don't really care since this was already way longer than 80 characters.